Firestore/Source/Public/FirebaseFirestore/FIRFirestore.h

```diff
@@ -103,7 +104,16 @@ NS_SWIFT_NAME(Firestore)
 #pragma mark - Configure FieldIndexes
 
 /**
- * This method is in preview. API signature and functionality are subject to change.
+ * A PersistentCacheIndexManager which you can config persistent cache indexes used for
+ * local query execution.
+ */
+@property(nonatomic, readonly, nullable)
+    FIRPersistentCacheIndexManager *persistentCacheIndexManager;
+
+/**
+ * NOTE: This preview method will be deprecated in a future major release, consider using
```

Comment:
Edit:

"NOTE: This preview method will be deprecated in a future major release. Consider using
`PersistentCacheIndexManager.enableIndexAutoCreation()` to let the SDK decide whether to create cache
 indexes for queries running locally."

Reply:
Apply same to line 127.